results-processor/main.py

```diff
@@ -115,27 +141,21 @@ def task_handler():
     params = flask.request.form
     # Mandatory fields:
     uploader = params['uploader']
-    gcs_path = params['gcs']
+    gcs_paths = params.getlist('gcs')
     result_type = params['type']
     # Optional fields:
     labels = params.get('labels', '')
 
-    # TODO(Hexcles): Support multiple results.
-    assert result_type == 'single'
-
-    match = re.match(r'/([^/]+)/(.*)', gcs_path)
-    assert match
-    bucket_name, blob_path = match.groups()
-
-    gcs = storage.Client()
-    bucket = gcs.get_bucket(bucket_name)
-    blob = bucket.blob(blob_path)
+    assert (
+        (result_type == 'single' and len(gcs_paths) == 1) or
```

Comment:
Nit: indentation is a bit odd. do we use autopep8 or something for formatting?

Reply:
Hmm, I use https://github.com/Vimjas/vim-python-pep8-indent , which thinks this is the right indentation. What do you think it should look like?

Reply:
It's just the outermost parens that make it look a bit funny, might have expected a pretty-printer to put `assert ((result_type == 'single' and len(gcs_paths) == 1) or` on the first line and indent the second. Doesn't matter, mostly curious if any style is enforced by checks in the repo itself.